packages/ui-components/src/components/DecisionTree/Column.js

```diff
@@ -0,0 +1,143 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+
+import React, { useCallback, useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import IconButton from '@material-ui/core/IconButton';
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemText from '@material-ui/core/ListItemText';
+import ChevronRightIcon from '@material-ui/icons/ChevronRight';
+import { ColumnFilter, useFilter } from './ColumnFilter';
+import { FetchLoader } from '../FetchLoader';
+
+const CLASS_NAME = 'TupaiaDecisionTreeColumn';
+
+const Container = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: stretch;
+`;
+
+const ContentContainer = styled.div`
+  flex: 1;
+  width: 248px;
+  padding: 12px;
+  overflow-y: auto;
+  border-right: 1px solid ${({ theme }) => theme.palette.grey['400']};
+
+  .MuiButtonBase-root {
+    border-radius: 10px;
+  }
+`;
+
+export const Column = ({ data, isLoading, error, showExpandIcon, onSelect, onExpand }) => {
+  const { updateFilter, applyFilter, checkMatchesFilter } = useFilter();
+  const [selected, setSelected] = useState(undefined);
+
+  const filteredData = applyFilter(data, { field: 'name' });
+  const isError = !!error;
+
+  useEffect(() => {
+    setSelected(undefined);
+  }, [data]);
```

Comment:
This means that we will the selected item when `data` changes , e.g. when a different entity is selected in an ancestor column